modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionList.tsx

```diff
@@ -41,10 +41,15 @@ const ActionList = ({
 			]}
 			className="mt-0 p-1"
 			creationMenuItems={[
-				{
-					label: Liferay.Language.get('add-action'),
-					onClick: createFDSAction,
-				},
+				noItemsButtonLabel === 'New Item Action'
```

Comment:
Where does this string come from? It seems like it could be easily broken if someone changes a lang key or something

Reply:
They are created here: https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Actions.tsx#L335-L355

They can be either `New Item Action` or `New Creation Action`.

I see your point on this if someone is not aware that this is dependent on the keys.

I could create a `creationMenuItemsLabel` so a mistake like this won't occur.